src/main/java/org/radarbase/management/service/UserService.kt

```diff
@@ -365,28 +322,7 @@ class UserService @Autowired constructor(
             user.password = encryptedPassword
             log.debug("Changed password for User: {}", user)
         }
-    }
 
-    /**
-     * Change the admin user's password. Should only be called in application startup
-     * @param email the new admin email
-     */
-    @Transactional
-    suspend fun addAdminEmail(email: String): UserDTO {
-        // find the admin user
-        val user = userRepository.findOneByLogin("admin")
-            ?: throw Exception("No admin user found")
-
-        // add the email
-        user.email = email
-        log.debug("Set admin email to: {}", email)
-
-        // there is no identity for this user, so we create it and save it to the IDP
-        val id = identityService.saveAsIdentity(user)
-        // then save the identifier and update our database
-        user.identity = id?.id
-        return userMapper.userToUserDTO(user)
-            ?: throw Exception("Admin user could not be converted to DTO")
     }
```

Comment:
Is there an alternative flow to get admin access? we should make sure to describe it, the current flow is in the readme I believe so it should be removed if not functional anymore

Reply:
After discussions, I'll just revert the changes.